src/components/ProgressSection.jsx

```diff
@@ -0,0 +1,27 @@
+import { ValidationStrategy as PropTypes } from "@eslint/object-schema";
+
+export default function ProgressSection({ applicationsCount, total }) {
+  const ratio = applicationsCount / total;
+
+  return (
+    <div className="mt-4">
+      <p className="text-base font-light mb-2">Progress</p>
+
+      <div className="h-6 w-full bg-darkBeige rounded-full overflow-hidden">
+        <div
+          className="h-full bg-accentOrange rounded-full"
+          style={{ width: `${Math.min(ratio * 100, 100)}%` }}
+        ></div>
+      </div>
+
+      <p className="mt-3 text-bordeaux text-center text-sm">
+        You&apos;ve applicationsCount to {applicationsCount} out of {total} jobs!
+      </p>
```

Comment:
The `index.css` file already sets the correct text size for the `p` tag
```suggestion
      <p className="mt-3 text-bordeaux text-center">
        You&apos;ve applied to {applicationsCount} out of {total} jobs!
      </p>
```